research/evm_explorer_comparison.md

```diff
@@ -0,0 +1,152 @@
+
+- **User Experience:**
+
+	The interface of Blockscout might not be as polished or intuitive as that of Etherscan. This can make it a bit harder to use, especially for those new to blockchain exploration. Developers might need to invest extra time in familiarizing themselves with the UI or improving it for end-users.
+
+- **Community Support Variability:**
+
+	The quality and availability of support can vary depending on the community’s activity and engagement. For developers, this means that while you can often get help from the community, there might be times when support is slower or less comprehensive compared to a commercial support service.
+
+## Comparative Analysis
+
+- **User Interface and Experience:**
+Etherscan's interface is sleek and user-friendly, making it a breeze to find the information you need. Its design is intuitive, so even newcomers can navigate it without much hassle. On the flip side, Blockscout, while fully functional, doesn't have the same level of polish. You might need a bit more time to get used to its layout and find what you're looking for.
+
+- **Data Availability and Accuracy:**
+Both Etherscan and Blockscout deliver detailed and accurate blockchain data. But because Etherscan has been around longer and focuses solely on Ethereum, it might have a slight edge in terms of data accuracy and reliability. Its established reputation adds an extra layer of trust for developers seeking precise blockchain information.
+
+- **APIs and Integrations:**
+Etherscan is well-known for its powerful APIs and developer tools, which is why it's a go-to for many developers working on Ethereum projects. These APIs are robust and well-documented, making it easy to integrate blockchain data into your applications. Blockscout also offers solid APIs, but its real strength is its open-source nature, which allows you to customize and extend its functionality to meet your specific needs.
+
+- **Customization and Extensibility:**
+Blockscout really stands out when it comes to customization and extensibility. Since it's open-source, you can modify it to match your exact requirements, adding new features or integrating it with other systems as needed. Etherscan, being a proprietary service, offers limited options for customization, which might be a drawback if you need a more tailored solution.
+
+- **Community and Support:**
+Etherscan has the advantage of a professional support team and extensive documentation, which can be a big help if you run into issues. Blockscout relies more on community support, which has its pros and cons. The quality of support can vary, but the community-driven approach often leads to innovative solutions and continuous improvements from various contributors.
```

Comment:
You keep stating blockscout only has community support? Is that true? Blockscout recently raised $3M seed round. I bet they also have a dedicated support. And by being a startup, their response rate might be better than Etherscan's